### ISSUE TITLE
Incorrect Primary Storage Usage Display

### ISSUE THREAD

**Issue by win98se** [torchgm/NewModernWinver#7]
![image](https://user-images.githubusercontent.com/11819816/156864017-6d42cab7-20f1-4b1b-afaa-ba2295acd0b7.png)
Seems like the used and free bars are inverted for primary storage.

**Comment by Sayor0630** [comment]
Yeah. They also can make the CPU, RAM usage more detailed but compact like this one.

**Comment by Gakubuchi** [comment]
Same issue here. It looks like the storage is almost full, but it is the other way around!
![modernwinver](https://user-images.githubusercontent.com/7223004/158987117-d517b5bf-e117-4f19-8e70-0fd8813cd98a.png)

**Comment by ElitePheonix2009** [comment]
Same Issue. It looks like more than 50% is used, but it's the other way around i.e. more than 50% is free

![image](https://user-images.githubusercontent.com/114980232/200596288-de15fec9-0074-4e6b-899d-b23d3af01b57.png)
